.github/workflows/triage.yml

```diff
@@ -0,0 +1,61 @@
+name: Add new issues to triage column
+
+on:
+  issues:
+    types:
+      - opened
+
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  ORGANIZATION: ethereum
+  REPOSITORY: solidity
+  PROJECT_NUMBER: 43
+  DRY_RUN: true
+
+jobs:
+  triage_issues:
+    runs-on: ubuntu-latest
+    if: join(github.event.issue.labels) == ''
+    steps:
+      - name: Retrieve the content of all columns on the board
+        run: |
+          gh api graphql \
+            --raw-field owner="$ORGANIZATION" \
```

Comment:
I'd change this to `${{ env.ORGANIZATION }}`. Same for other variable references below.

Reply:
Why though? These are just environment variables. I think it's fine to use them like this (and it looks cleaner :)).

I'd actually do the opposite and replace `"${{ env.DRY_RUN }}"` below with `$DRY_RUN` :)

Reply:
I replaced the `"${{ env.DRY_RUN }}"` because it was indeed wrong.
Just to clarify the use of `env` in the github actions, it is used to define the scope of the environment variable in the workflow, and it can be accessed directly, e.g. `$SOMETHING`, using the appropriate syntax of the runner environment (i.e. Ubuntu with bash, in our case), or accessed through the equivalent context property, e.g. `${{ env.SOMETHING }}`. According to the docs, the latter is designed to be used in the parts of the workflow that is not processed by the runner but by the GitHub Actions itself. So, using the `DRY-RUN` through context was indeed not the recommended way.